docs/examples/image_binarization/image_binarization.jl

```diff
@@ -0,0 +1,128 @@
+# ---
+# title: Image Binarization
+# author: Ashwani Rathee
+# date: 2021-7-2
+# ---
+
+# ImageBinarization.jl provides several algorithms for binarizing images
+# into background and foreground(bi-level image). In this demonstration, 
+# we'll be exploring where these algorithms could be useful and learn how
+# different algorithms are suited for different types of tasks.
+
+# Suppose a person wants to make an automatic Sudoku solver using Computer 
+# Vision . Before anything else that person needs to import the image, 
+# preprocess it to optimize it for cell content extraction which can then 
+# be used for solving the sudoku. 
+
+# Let's first import packages and then testimage sudoku
+
+using Images, ImageBinarization, TestImages
+using  CoordinateTransformations, Rotations, ImageTransformations
+
+# Original Image
+
+img = testimage("sudoku")
+
+# Grayscale version as binarize! only accepts Grayscale images
+
+img = Gray.(img)
+
+# Let's first rotate our image
+
+trfm = recenter(RotMatrix(pi / 30), center(img));
+imgw = warp(img, trfm)
+
+# Now let's zoom in a bit
+
+imgw = parent(imgw)[65:490, 65:490]
```

Comment:
For this, we should instead introduce `imrotate(img, degree, indices)` API

Reply:
Sure, we need to add imrotate example here too:https://juliaimages.org/latest/pkgs/transformations/

Reply:
Actually, it's already supported in ImageTransformations, e.g., `imrotate(img, 0.3, axes(img))`.a

The transformation documentation should be removed and redirected to our new one: https://juliaimages.org/ImageTransformations.jl/dev